pyaedt/edb.py

```diff
@@ -570,6 +590,12 @@ def core_padstack(self):
             self._padstack = EdbPadstacks(self)
         return self._padstack
 
+    @property
+    def core_padstack_instance(self):
+        if not self._padstack_instance and self.builder:
```

Comment:
@ring630 Is it `self._padstack_instances` or `self._padstack_instance` here because you are instantiating `EdbPadstackInstances` and not `EdbPadstackInstance`.

Reply:
The idea is self._padstack_instance is instantiated from EdbPadstackInstances which has a list of EdbPadstackInstance.

Reply:
Done